device-data/types/basal/suspend.md

```diff
@@ -141,6 +141,85 @@ See [`previous`](./previous.md) for detailed documentation on this deprecated fi
 	Required:
 		jellyfish: no (optional)
 		platform: no (optional)
+May contain—only!—the following properties:
+
+ * type
+ * deliveryType
+ * percent
+ * rate
+ * scheduleName
+ * suppressed
+
+#### suppressed.type
+
+[ingestion, storage, client] The string `basal`.
+
+	QUICK SUMMARY
+	Required:
+		jellyfish: no (optional)
+		platform: yes
+
+#### suppressed.deliveryType
+
+[ingestion, storage, client] A string—either `scheduled` or `temp`—encoding the `deliveryType` of the currently suppressed basal event.
+
+	QUICK SUMMARY
+	Required:
+		jellyfish: no (optional)
+		platform: yes
+	Range: Must be one of:
+		`scheduled`
+		`temp`
+
+#### suppressed.percent
+
+> This field is **optional**.
+
+[ingestion, storage, client] A floating point number >= 0 representing a percentage multiplier of the current basal rate to obtain the temp rate in units per hour.
+
+	QUICK SUMMARY
+	Required:
+		jellyfish: no (optional)
+		platform: no (optional)
+	Numerical type: Floating point value representing a percentage, where 1.0 represents 100%.
+	Range:
+		min: 0.0
+		max: 10.0
```

Comment:
What's the question? We represent 0 - 100% as a float from 0 to 1.0, so 0.1 is 10% etc.

The reason it goes to 10.0 is people can and often do program `temp`s with percentages *greater* than 100%.